client/my-sites/domains/domain-management/name-servers/style.scss

```diff
@@ -6,14 +6,17 @@
 
 .name-servers__dns {
 	.name-servers__toggle {
-		display: inline-block;
-		float: right;
-
-		.components-toggle-control .components-base-control__field .components-form-toggle {
+		.components-base-control__field {
+			justify-content: space-between;
 			margin-right: 0;
```

Comment:
The right margin here won't do the job it did before. Note that with the latest change, `margin-right: 0;` is now declared for the `.components-base-control__field` selector, while previously it was for the `.components-toggle-control .components-base-control__field .components-form-toggle` selector.